src/Model/Generated/Endpoint/Avatar.php

```diff
@@ -34,7 +34,8 @@ class Avatar extends BunqModel
     /**
      * Object type.
      */
-    const OBJECT_TYPE = 'Avatar';
+    const OBJECT_TYPE_POST ='Uuid';
```

Comment:
No space after `=`, while it used to be there. Check the generator?

Reply:
Ohhhh farts :see_no_evil: